### ISSUE TITLE
[ACCESSIBILITY] Make the current position of the focus clearly

### ISSUE THREAD

**Issue by markusmann-vg** [SORMAS-Foundation/SORMAS-Project#9643]
### Feature Description
In many areas, the focus is not visible. In the menu, no marker is visible when focus is active. This can be adjusted via CSS.

The markings with active focus of buttons and input fields (e.g. button "New case" or field "Case origin") are very unclear (only a slight shadow). This marking should be more obvious. The following measures can be taken: Frames, underlining, color inversion, shape changes or additional markings.

When text fields receive focus, a vertical bar is displayed in the field, indicating that the user can insert text, OR all of the text is highlighted, indicating that the user can type over the text.
When a user interface control receives focus, a visible border is displayed around it.

### Proposed Change
The keyboard focus should be highlighted at least as clearly as the mouse focus.

For links that do not respond to the mouse pointer, the keyboard focus should at least stand out clearly from the selected background color.

- Focus Visible => Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

### Acceptance Criteria
How to check:
1. Tab through all controls and check if they react with graphical changes to the focus (for example with color changes, underlining or faded in symbols). Hidden jump links should be shown when focus is maintained.
2. If focus highlighting is done solely by a color change, check that the contrast ratio between focused and unfocused is at least 3:1.
3. For links that do not change graphically, check whether they react to the mouse pointer.
4. If this is the case: cancel or continue with 6.
5. If only the standard browser keyboard focus (system wreath) appears, check whether it is easily recognizable on it in front of designed (e.g. colored via CSS) backgrounds.
6. If in doubt, determine whether the contrast ratio between system focus and background is at least 3:1 according to test step 9.1.4.3 "Text contrasts sufficient".
7. Load page in Chrome browser and repeat steps 1-6.

### Implementation Details

[G149: Using user interface components that are highlighted by the user agent when they receive focus](https://www.w3.org/WAI/WCAG21/Techniques/general/G149)
[C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
[G165: Using the default focus indicator for the platform so that high visibility default focus indicators will carry over](https://www.w3.org/WAI/WCAG21/Techniques/general/G165)
[G195: Using an author-supplied, visible focus indicator](https://www.w3.org/WAI/WCAG21/Techniques/general/G195)
[C40: Creating a two-color focus indicator to ensure sufficient contrast with all components](https://www.w3.org/WAI/WCAG21/Techniques/css/C40)
[SCR31: Using script to change the background color or border of the element with focus](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR31)

### Additional Information
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html
